umei-openapi.json

```diff
@@ -1453,6 +1413,35 @@
       }
     },
     "parameters": {
+      "take": {
+        "name": "take",
+        "in": "query",
+        "description": "Number of hits to return.  If client does not specify Take the server MAY apply a default Take value (which will be returned in the SearchResult object). The default value is not guaranteed to be the same for different requests\"",
+        "schema": {
+          "type": "integer",
+          "description": "Number of hits to return.  If client does not specify Take the server MAY apply a default Take value (which will be returned in the SearchResult object). The default value is not guaranteed to be the same for different requests\"",
+          "format": "int32"
+        }
+      },
+      "skip": {
+        "name": "skip",
+        "in": "query",
+        "schema": {
+          "type": "integer",
+          "format": "int32"
+        }
+      },
+      "orderBy": {
+        "name": "orderBy",
+        "in": "query",
+        "schema": {
+          "type": "array",
+          "items": {
```

Comment:
Could you maybe add an example for the "orderBy" parameter, and explain the values allowed ? Do we need to return a specific error if the string provided is not valid ?